src/main/resources/assets/opencomputers/lang/de_DE.lang

```diff
@@ -424,9 +424,9 @@ achievement.oc.rack.desc=Wird unter Anderem in Kantinen genutzt, um gebrauchte T
 achievement.oc.raid=Suche Team
 achievement.oc.raid.desc=Heroic plzkthx.
 achievement.oc.ram=Random Access Memories
-achievement.oc.ram.desc=Congratulations, you're Doin' It Right.
+achievement.oc.ram.desc=Gratuliere! Du machst es richtig.
```

Comment:
"Doin' It Right" ist soweit ich weiß eine Anspielung auf Daft Punk, daher habe ich das nicht übersetzt.